src/fsharp/LowerCallsAndSeqs.fs

```diff
@@ -483,31 +483,33 @@ let ConvertSequenceExprToObject g amap overallExpr =
         // transferred to the r.h.s. are not yet compiled.
         //
         // TODO: remove this limitation
```

Comment:
can the comment be removed?

Reply:
Yes, though I need to make sure we have a test case for that specific case.  I'll note it in the TODO list above